tools/benchmark/generate_trace_models.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import com.datadog.gradle.plugin.apisurface.ApiSurfacePlugin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+val generateTraceModelsTaskName = "generateTraceModelsFromJson"
+
+tasks.register(
+    generateTraceModelsTaskName,
+    com.datadog.gradle.plugin.jsonschema.GenerateJsonSchemaTask::class.java
+) {
+    inputDirPath = "src/main/json"
+    targetPackageName = "com.datadog.benchmark.internal.model"
+}
+
+afterEvaluate {
+    tasks.findByName(ApiSurfacePlugin.TASK_GEN_KOTLIN_API_SURFACE)
+        ?.dependsOn(generateTraceModelsTaskName)
```

Comment:
There is no public API generation task in this module, this can be removed.

Reply:
removed